test/e2e/common.go

```diff
@@ -87,4 +91,5 @@ func newSecret(namespace string, name string, data map[string][]byte) *corev1.Se
 // CloudAssert defines assertions to perform on the cloud provider.
 type CloudAssert interface {
 	HasPodVM(t *testing.T, id string) // Assert there is a PodVM with `id`.
+	CmdsToCheckConfidentialPodVM(pod *corev1.Pod) []testCommand
```

Comment:
We're adding a new assert method in `CloudAssert`, shall it be more general like `HasConfidentialPodVM`?